### PR TITLE
Feature/entry review remove approve verify 2

### LINKED ISSUE #1527: Entry Review: Remove approval and verify buttons radio buttons from entry review modal

### LINKED ISSUE #1528: Entry Review: Rename approval to verify

### LINKED ISSUE #1530: Entry Review: Change labels for verify and control status change along with its status

### LINKED ISSUE #1532: Entry Review: Hide control status for non QC users

### PR DESCRIPTION
**Pull request by AdityaKhatri**
fixes https://github.com/the-deep/client/issues/1529
fixes #1528 
fixes #1527 
fixes #1530 
fixes #1532 
depends on https://github.com/the-deep/server/pull/746

### CODE REVIEW

**Review by frozenhelium — APPROVED** [review]

File: src/redux/initial-state/dev-lang.json [hunk 1]
```diff
@@ -1336,44 +1338,43 @@
...
+        "-15568595": "Verified by you and {userCount} other users",
+        "-4503674": "Not verified",
+        "-44367329": "Not verified by you (Verified by {userCount} other users)",
+        "-46401322": "{n} peer verification(s)",
         "-52742063": "comment",
         "-97072562": "{assigner} assigned you to a new {comment}.",
-        "-56701570": "{assigner} approved an {entry} in project {projectName}.",
-        "-10980487": "{assigner} verified an {entry} in project {projectName}.",
-        "-96661189": "{assigner} unverified an {entry} in project {projectName}.",
+        "-56701570": "{assigner} verified aREvyn {entry} in project {projectName}.",
```
Comment (tnagorra) [inline src/redux/initial-state/dev-lang.json]:
```suggestion
        "-56701570": "{assigner} verified an {entry} in project {projectName}.",
```

File: src/redux/initial-state/dev-lang.json [hunk 2]
```diff
@@ -1188,17 +1190,17 @@
...
         "-72741271": "Showing all entries",
         "-55557162": "This export is currently in queue to be exported and has not been started.",
-        "-11119478": "{verified} out of {total} entries are verified in this category.",
+        "-11119478": "{controlled} out of {total} entries are verified in this category.",
         "-79879872": "Expand All",
         "-79797971": "Collapse All",
         "-46164024": "Jump to",
         "-19695002": "Show table of contents",
-        "-81087776": "{verifiedCount} verified out of {entriesCount} entries",
+        "-81087776": "{controlledCount} verified out of {entriesCount} entries",
```
Comment (tnagorra) [inline src/redux/initial-state/dev-lang.json]:
```suggestion
        "-81087776": "{controlledCount} controlled out of {entriesCount} entries",
```

File: src/redux/initial-state/dev-lang.json [hunk 3]
```diff
@@ -1188,17 +1190,17 @@
...
         "-987979871": "Sources",
-        "-879879798": "Unverified entries",
-        "-897879798": "Verified entries",
+        "-879879798": "Uncontrolled entries",
+        "-897879798": "Controlled entries",
         "-978987098": "Showing entries based on ",
         "-72741271": "Showing all entries",
         "-55557162": "This export is currently in queue to be exported and has not been started.",
-        "-11119478": "{verified} out of {total} entries are verified in this category.",
+        "-11119478": "{controlled} out of {total} entries are verified in this category.",
```
Comment (tnagorra) [inline src/redux/initial-state/dev-lang.json]:
```suggestion
        "-11119478": "{controlled} out of {total} entries are controlled in this category.",
```

File: src/components/general/ToggleEntryControl/index.tsx [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+        url: `server://v2/projects/${projectId}/project-memberships/`,
+        method: 'GET',
+        autoTrigger: true,
+        onFailure: notifyError(_ts('entryReview', 'projectMemberList')),
+    });
+
+    const isQualityController = useMemo(() =>
+        projectMembersResponse?.results.find(
+            v => v.member === activeUser.userId,
+        )?.badges.some(v => v === 0),
```
Comment (tnagorra) [inline src/components/general/ToggleEntryControl/index.tsx]:
```suggestion
        )?.badges.some(v => v === SOME_ENUM_HERE),
```

**Review by tnagorra — CHANGES_REQUESTED** [review]
Some problem with translations

**Review by tnagorra — APPROVED** [review]

**Pull request merged**